Userland/Libraries/LibCore/EventLoop.h

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
  * Copyright (c) 2022, kleines Filmröllchen <malu.bertsch@gmail.com>
+ * Copyright (c) 2022, Jelle Raaijmakers <jelle@gmta.nl>
```

Comment:
Adding yourself to the copyright header for a typo fix seems a bit... uh, extreme? :^)

Reply:
Since I was going to add it somewhere in this PR, it technically seemed better to me to do it at the _first_ change instead of what would be the subjective "significant enough" change. I don't want these changes to lead the conversation here though, so I've removed them.